### PR TITLE
refactor(cdk): move old control class to legacy

### PR DESCRIPTION
**Pull request by waterplea**
Closes #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/kit/components/input-phone-international/input-phone-international.component.ts [hunk 1]
```diff
@@ -163,7 +134,6 @@ export class TuiInputPhoneInternationalComponent
     public onItemClick(isoCode: TuiCountryIsoCode): void {
         this.open = false;
         this.countryIsoCode.set(isoCode);
-        this.nativeFocusableElement?.focus();
```
Comment (nsbarsukov) [inline projects/kit/components/input-phone-international/input-phone-international.component.ts:166]: Why was this line removed ?

**Review by nsbarsukov — APPROVED** [review]

File: projects/kit/components/input-phone-international/input-phone-international.component.ts (hunk 1 above)
Comment (waterplea) [reply projects/kit/components/input-phone-international/input-phone-international.component.ts:166]: Fixed
